### PR TITLE
Add hero 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- There is no Hero section. #5 
- A logo has been implemented. #6 

**What is the new behavior (if this is a feature change)?**
- A Hero section has been added. #5 
- A logo has been implemented. #6

### CODE REVIEW

File: src/components/organisms/Hero/index.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ MDBRow,
+ MDBCol,
+ MDBBtn,
+ MDBIcon,
+ MDBAnimation,
+} from 'mdbreact';
+
+//> Images
+// Blue logo of Blue Lupi
+import lupiLogoWhite from '../../../assets/logo_white_text.png';
```
Comment (Aichnerc) [inline src/components/organisms/Hero/index.jsx]: This comment is not suitable. (white instead of blue)

**Review by Aichnerc — COMMENTED** [review]
One more change spotted by my site.

File: src/components/organisms/Hero/index.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ MDBMask,
+ MDBRow,
+ MDBCol,
+ MDBBtn,
+ MDBIcon,
+ MDBAnimation,
+} from 'mdbreact';
+
+//> Images
+// Blue logo of Blue Lupi
```
Comment (kleberbaum) [inline src/components/organisms/Hero/index.jsx]: Is it supposed to be blue or white.

File: src/utilities/_fonts.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@font-face {
+    font-family: 'Heavitas';
+    src: url('utilities/fonts/heavitas-webfont.woff2') format('woff2'),
+         url('utilities/fonts/heavitas-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+
```
Comment (kleberbaum) [inline src/utilities/_fonts.scss]: whitespace?

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/organisms/Hero/index.jsx (hunk 2 above)
Comment (Aichnerc) [reply src/components/organisms/Hero/index.jsx]: Has been fixed

File: src/utilities/_fonts.scss (hunk 3 above)
Comment (Aichnerc) [reply src/utilities/_fonts.scss]: Whitespace . . . . . . . . . . . .  is . . . . . . . . . . . . . . nice.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**